test2/out2.list

```diff
@@ -1,3 +1,4 @@
+@0
```

Comment:
Why change this? Shouldn't we keep it consistent with the output of the assembler?

Reply:
I wanted to make it more convenient to change the instruction.img, so I added the entire format